modules/data.land/R/cohort2pool.R

```diff
@@ -1,56 +1,103 @@
 ##' cohort2pool function
 ##'Calculates total biomass using veg cohort file. 
-##'
+##' @name cohort2pool
+##' @title cohort2pool
+##' @description Converts .rds files into pool netcdf files.
 ##' @export
+##'
 ##' @param veg_file path to standard cohort veg_file
+##' @param dbh_name Default is "DBH". This is the column name in the veg_file that represents DBH. May differ depending on data source.
 ##' @param allom_param parameters for allometric equation, a and b. Based on base-10 log-log linear model (power law)
+##'
 ##' @author Saloni Shah
-##' @examples
+##'
 ##' \dontrun{
 ##' veg_file <- "~/downloads/FFT_site_1-25665/FFT.2008.veg.rds"
 ##' cohort2pool(veg_File = veg_file, allom_param = NULL)
 ##' }
 
-cohort2pool <- function(veg_file, allom_param = NULL) {
+cohort2pool <- function(veg_file, allom_param = NULL, dbh_name="DBH") {
   
   ## Building Site ID from past directories
-  path <- dirname(veg_file)
-  last_dir <- basename(path)
-  nums_id <- strsplit(last_dir,"[^[:digit:]]")
-  base_id <- nums_id[[1]][length(nums_id[[1]])]
-  suffix <- nums_id[[1]][(length(nums_id[[1]])-1)]
-  siteid = as.numeric(suffix)*1e9 + as.numeric(base_id)
-  
+  # path <- dirname(veg_file)
+  # last_dir <- basename(path)
+  # nums_id <- strsplit(last_dir,"[^[:digit:]]")
+  # base_id <- nums_id[[1]][length(nums_id[[1]])]
+  # suffix <- nums_id[[1]][(length(nums_id[[1]])-1)]
+  #siteid = as.numeric(suffix)*1e9 + as.numeric(base_id)
+  siteid = 646 #Need to manually set when running line-by-line, gets siteid from veg_file filepath
+  outdir = "/projectnb/dietzelab/ahelgeso/NEON_ic_data/Harvard/neon_nc_ens/"
```

Comment:
Site and outdir shouldn't be hard coded. More generally, these changes seem like they should have been in a different PR